cmd/rook/object/create.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package object
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/rook/rook/cmd/rook/rook"
+	"github.com/rook/rook/pkg/rook/client"
+	"github.com/rook/rook/pkg/util/flags"
+	"github.com/spf13/cobra"
+)
+
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Creates a new object storage instance in the cluster",
+}
+
+func init() {
+	createCmd.RunE = createObjectStoreEntry
+}
+
+func createObjectStoreEntry(cmd *cobra.Command, args []string) error {
+	rook.SetupLogging()
+
+	if err := flags.VerifyRequiredFlags(cmd, []string{}); err != nil {
```

Comment:
for some reason, i've been under the impression that this call will check default params up stream too (such as api-client-endpoint), but it does not do that.  this call is not required, it's essentially a no-op.